pulpcore/pulpcore/app/serializers/repository.py

```diff
@@ -411,10 +411,19 @@ class RepositoryVersionSerializer(ModelSerializer, NestedHyperlinkedModelSeriali
         help_text=_('A list of content units to remove from the latest repository version'),
         write_only=True
     )
+    base_version = serializers.HyperlinkedRelatedField(
+        required=False,
+        help_text=_('A repository version whose content will be used as the initial set of content '
+                    'for the new repository version'),
+        queryset=models.RepositoryVersion.objects.all(),
+        view_name='repository-versions-detail',
+        label=_('Base Repository Version'),
+        write_only=True
```

Comment:
Yea, it's not a field on the model. I'm a bit torn about whether it should be. It would help in cases of debugging to know if there was a base version. However, it makes thing a bit more complex (ie what if the base version gets deleted?). There was talk about even storing all the params used to produce a repo version somehow.

Reply:
Thought about it some and will add it as a model field.

Reply:
+1 should be a foreign key imo